h/static/scripts/group-forms/components/test/CreateGroupForm-test.js

```diff
@@ -116,6 +116,44 @@ describe('CreateGroupForm', () => {
     });
   });
 
+  describe('name field', () => {
```

Comment:
These are general tests for the `TextField` component as much as the name field. I expect it will make sense to extract that into its own module soon.

Reply:
Yep